modules/tensor_mechanics/doc/content/modules/tensor_mechanics/ShellElements.md

```diff
@@ -0,0 +1,78 @@
+# Shell elements
+
+A shell element is used to model the response of structural elements that are much thinner along one direction (out-of-plane direction) compared to the two in-plane directions. A 4-node shell element is implemented in Moose based on [!cite](dvorkin1984continuum) that can model structural response of both thin and thick plates.
+
+!media media/tensor_mechanics/shell_representation.png
+      style=width:50%;margin-left:2%;float:right
+      id=fig_shell
+      caption=Shell element with 4 nodes and 3 translational and 2 rotational degrees of freedom at each node.
```

Comment:
Looks like this image was pulled from that paper. We should at a minimum say it was taken from that paper, but to be safe, we probably shouldn't even include the image. I'd say for now, let's pull it out, and maybe make our own image.

Reply:
I have added a different figure.